opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -373,14 +373,14 @@ message HistogramDataPoint {
   // This defines size(bounds) + 1 (= N) buckets. The boundaries for bucket
   // at index i are:
   //
-  // [0, bounds[i]) for i == 0
+  // (-infinity, bounds[i]) for i == 0
   // [bounds[i-1], bounds[i]) for 0 < i < N-1
   // [bounds[i], +infinity) for i == N-1
   // The values in bounds array must be strictly increasing and > 0.
```

Comment:
The requirement `and > 0` should probably be removed, since with `-infinity` lower bound there could be negative values?

Reply:
Good catch, thanks! Removed.